src/main.rs

```diff
@@ -318,13 +319,28 @@ async fn main() -> anyhow::Result<()> {
         policy_changes_topic: args.policy_changes_topic,
         policy_timeout_ms: args.policy_timeout_ms,
     };
-    let labels = args
+    let mut labels = args
         .label
         .unwrap_or_default()
         .iter()
         .map(|labelpair| parse_label(labelpair))
         .collect::<anyhow::Result<HashMap<String, String>, anyhow::Error>>()
         .context("failed to parse labels")?;
+    let labels_from_args: HashSet<String> = labels.keys().cloned().collect();
+    labels.extend(env::vars().filter_map(|(key, value)| {
+        let key = if key.starts_with("WASMCLOUD_LABEL_") {
+            key.strip_prefix("WASMCLOUD_LABEL_")?.to_string()
+        } else {
+            return None;
+        };
+        if labels_from_args.contains(&key) {
+            warn!(
+                ?key,
+                "label provided via environment variable will override label set via args"
+            );
+        }
+        Some((key, value))
```

Comment:
Is this the common order of precedence that we normally take? e.g. 
```
WASMCLOUD_LABEL_foo=bar ./wasmcloud --label foo=baz
```
This code reads like `bar` would be the value of `foo`, but I would argue that we should take the arg value over the environment variable. It may be confusing to have something like 
```
# ~/.zshrc
export WASMCLOUD_LABEL_me=$(whoami)
# later
wash up --label me=test
```

I'm good either way, so long as we're consistent, and I lean towards the arg taking precedence as it's explicitly provided at runtime

Reply:
Nope, you're right, I got this backward. Fixed!